### PR TITLE
Be graph endpoint

### PR DESCRIPTION
**Pull request by nhistory**
- Added /api/car/graph/{maker} endpoint for Chart.js

### CODE REVIEW

File: app/services/database.py [hunk 1]
```diff
@@ -30,6 +30,41 @@ async def fetch_car_by_maker(maker: str) -> list:
...
+        filtered_data = []
+        for data in data_list:
+            if data[0] == model:
+                filtered_data.append({
+                    'year': data[1],
+                    'price': data[2],
+                    'mileage': data[3]
+                })
+        filtered_list.append(filtered_data)
+    print(filtered_list)
```
Comment (aurora-yn) [inline app/services/database.py]: Please remove print statement :)

File: app/services/database.py [hunk 2]
```diff
@@ -30,6 +30,41 @@ async def fetch_car_by_maker(maker: str) -> list:
...
+    model_list = list(set(model_list))
+
+    filtered_list = []
+    for model in model_list:
+        filtered_data = []
+        for data in data_list:
+            if data[0] == model:
+                filtered_data.append({
+                    'year': data[1],
+                    'price': data[2],
```
Comment (aurora-yn) [inline app/services/database.py]: price seems `data[3]`, and mileage is `data[2]` based on LN:41

File: app/services/database.py (hunk 1 above)
Comment (nhistory) [reply app/services/database.py]: I removed it. Thanks!

File: app/services/database.py (hunk 2 above)
Comment (nhistory) [reply app/services/database.py]: Nice catch! I fixed that index. Thanks!

**Review by nhistory — COMMENTED** [review]
Resolved issues from aurora-yn's comment

**Pull request merged**